alphagov/paas-cf
# [#110234556] Initiate CF deploy from Concourse

**What**

We want to merge the existing manifests from [cf-terraform](https://github.com/alphagov/cf-terraform) into this repository, including full history

**How this PR should be reviewed**

This PR should reviewed in the following context:

* I can see the full history from cf-terraform/cf-manifest directory in the manifests/cf-manifest directory of this repo
* I understand the git commands contained in [this gist](https://gist.github.com/timmow/94b0b6b89d0d394e110b) used to filter the branch

**How to test this PR**
Check the branch out and look at the history

It might help to run through the steps in [this gist](https://gist.github.com/timmow/94b0b6b89d0d394e110b) in order to help understand what is happening

**Who should review this PR**

* Anyone but me
